### ISSUE TITLE
Demo: auto-select offline copy of asset after storing

### ISSUE THREAD

**Issue by beaufortfrancois** [google/shaka-player#996]
**navigator.userAgent**
"Mozilla/5.0 (Linux; Android 8.0.0; Pixel Build/OPR6.170623.012) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/62.0.3194.0 Mobile Safari/537.36"

**Steps to reproduce:**
1. Go to https://nightly-dot-shaka-player-demo.appspot.com/demo/#asset=//storage.googleapis.com/shaka-demo-assets/angel-one-widevine/dash.mpd;lang=en-US
2. Click "Load" button
3. Expand "Offline section" and click "Store" button
4. Wait for asset to be fully downloaded.
5. Select another asset and click "Load" button
6. Turn ON airplane mode (meaning no internet connection)
7. Select back "Angel One (multicodec, multilingual, Widevine)" asset\
8. Click "Load" button

**Expected result:**
Video plays.

**What happens instead:**
I get the error below:
```
Shaka Error DRM.LICENSE_REQUEST_FAILED (shaka.util.Error {
  "severity": 2,
  "category": 1,
  "code": 1002,
  "data": [
    "http://widevine-proxy.appspot.com/proxy"
  ],
  "handled": false,
  "message": "Shaka Error NETWORK.HTTP_ERROR (http://widevine-proxy.appspot.com/proxy)",
  "stack": "Error: Shaka Error NETWORK.HTTP_ERROR (http://widevine-proxy.appspot.com/proxy)\n    at new shaka.util.Error (http://localhost:4000/lib/util/error.js:66:13)\n    at XMLHttpRequest.xhr.onerror (http://localhost:4000/lib/net/http_plugin.js:91:14)"
})
```

**Comment by beaufortfrancois** [comment]
And I've just realized an "Offline asset" has been added to the dropdown list...
I believe this makes my issue WAI.

**Comment by beaufortfrancois** [comment]
I'm closing this issue and updating the existing PR: https://github.com/google/shaka-player/pull/995

**Comment by joeyparrish** [comment]
I'm glad you figured it out, but it should be more obvious.  For example, we could automatically select the offline copy in the list after storing it.  I'm going to reopen this and schedule it for v2.3, but a demo app usability improvement like this could also be cherry-picked to v2.2.x releases.

**Comment by joeyparrish** [comment]
The fix has been cherry-picked to v2.2.x and will be released in v2.2.2.
